api/modules/currency/currency_item.py

```diff
@@ -0,0 +1,8 @@
+class CurrencyItem(object):
```

Comment:
Add docs

Reply:
What sort of docs do I really need to add,can you please tell?

Reply:
https://github.com/project-travel-mate/server/blob/master/api/modules/weather/weather_response.py